### ISSUE TITLE
.Net: Bug: Invalid Image URL - Requires HTTP/HTTPS or Base64 Data URL

### ISSUE THREAD

**Issue by Cobra86** [microsoft/semantic-kernel#6674]
**Describe the bug**
I'm encountering an "Invalid image URL" error when passing image data as bytes instead of a URI.

**To Reproduce**
This issue occurs when making a basic call to the Azure OpenAI API, passing userMessage and the image in bytes.

Code:
```
 public async Task<String> SendMessage(string userMessage)
 {
     var imageBytes = File.ReadAllBytes("C:\\MyProjects\\free-nature-images.jpg");

     var messages = new ChatMessageContentItemCollection
     {
         new TextContent(userMessage),
         new ImageContent(imageBytes)
     };

     Messages.AddUserMessage(messages);

     try
     {
         ChatMessageContent chatMessage = await _kernel.Services.GetService<IChatCompletionService>().GetChatMessageContentAsync(Messages, _openAIPromptExecutionSettings, _kernel);

         if (!String.IsNullOrWhiteSpace(chatMessage.Content))
         {
             Messages.Add(chatMessage);

             return chatMessage.Content;
         }

     }
     catch (Exception e)
     {
         return $"Error:{e.Message}";
     }

     return null;
 }
```
Result
```
User > hi

Assistant > Error:Invalid image URL. The URL must be a valid HTTP or HTTPS URL, or a data URL with base64 encoding.
Status: 400 (Bad Request)
ErrorCode: BadRequest

Content:
{"error":{"code":"BadRequest","message":"Invalid image URL. The URL must be a valid HTTP or HTTPS URL, or a data URL with base64 encoding.","param":null,"type":null}}
```

**Expected behavior**
I'm unsure if I'm passing the correct value. I'd expect to be able to pass a byte[] instead of a URL.

**Platform**
 - OS: Windows 11
 - IDE: Visual Studio
 - Language: C#
 - Source: Microsoft.SemanticKernel 1.14.1
 - AI Model: Azure OpenAI GPT 4o

**Additional context**

**Comment by Cobra86** [comment]
Apologise, I cloned the latest main branch and tested it and looks like it's already fixed. 

```
 public async Task<String> SendMessage(string userMessage)
 {
     var imageBytes = File.ReadAllBytes("C:\\MyProjects\\free-nature-images.jpg");

     var messages = new ChatMessageContentItemCollection
     {
         new TextContent(userMessage),
         new ImageContent(new ReadOnlyMemory<byte>(imageBytes), "image/jpeg")
     };

     Messages.AddUserMessage(messages);

     try
     {
         ChatMessageContent chatMessage = await _kernel.Services.GetService<IChatCompletionService>().GetChatMessageContentAsync(Messages, _openAIPromptExecutionSettings, _kernel);

         if (!String.IsNullOrWhiteSpace(chatMessage.Content))
         {
             Messages.Add(chatMessage);

             return chatMessage.Content;
         }

     }
     catch (Exception e)
     {
         return $"Error:{e.Message}";
     }

     return null;
 }
```

**Comment by gmantri** [comment]
I am wondering if there is an ETA for the new release that contains a fix for this issue. Thanks.

**Comment by dmytrostruk** [comment]
Hi @gmantri , we released version `1.15.0` today which should include this fix:
https://www.nuget.org/packages/Microsoft.SemanticKernel/1.15.0
